pkgs/top-level/all-packages.nix

```diff
@@ -33023,6 +33023,11 @@ with pkgs;
     jre = openjdk11;
   };
 
+  faiss = callPackage ../development/libraries/science/math/faiss {
+    pythonPackages = python3Packages;
+    swig = swig4;
```

Comment:
Is there anything special about swig4? Why not just let it use the default swig?

Reply:
Yes, swig3 doesn't support `-doxygen` option that upstream hardcodes

Reply:
gotcha, could we add a comment to that effect here?